cli/cmd/list_cmd.go

```diff
@@ -45,5 +59,9 @@ var listCmd = &cobra.Command{
 }
 
 func init() {
+	listCmd.PersistentFlags().Int32VarP(&listTake, "take", "t", 20, "Take number")
```

Comment:
Again, should these flags be persistent or not? Also, we can probablly skip the short version